docs/docusaurus/docs/core/installation_and_setup/manage_data_contexts.md

```diff
@@ -181,12 +181,9 @@ If you're using GX for multiple projects, you might want to use a different Data
 
 ### Prerequisites
 
-<Prerequisites requirePython = {false} requireInstallation = {true} requireDataContext = {false} requireSourceData = {null} requireDatasource = {false} requireExpectationSuite = {false}>
-
+- A Great Expectations instance.
```

Comment:
For now you can use the following format:
```suggestion
- [A Great Expectation instance](/core/installation_and_setup/install_gx.md).
```
In the future this doc will be revised to be in line with the format we settled on for the "Manage Expectations" topic -- we'll update it to use the standardized, imported prerequisites then.